dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -135,6 +146,12 @@ public void show() {
       return;
     }
     onAttached(rootView);
+    if(Build.VERSION.SDK_INT > 10){
```

Comment:
Another thing, I think this animation should be optional and/or customisable as not all users would like this animation. What do you think @orhanobut?